src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -160,12 +161,14 @@ public void islDefaultRulesDelete(Endpoint source, Endpoint destination) {
         emit(STREAM_SPEAKER_RULES_ID, getCurrentTuple(), makeIslRulesDeleteTuple(source, destination));
     }
 
+    // FIXME(surabujin) - storm handler design violation
```

Comment:
Didn't get it. Add some details please.

Reply:
Original design supposes for storm bolt to have `makeStreamNameTuple` methods for each stream. Here we define 2 such methods for 1 stream.